doc/ADVANCED.md

```diff
@@ -31,6 +31,9 @@ In order to retrigger a specific test, you need to add a comment to the PR.
 **PLESE NOTE**: gitarro will delete the comment where you write the retriggering command,
 so make sure that you use a separate comment for this.
 
+**NEW!**: You can also create a checkbox in the PR description with the text : Re-run test "<test_name>", 
```

Comment:
`NEW` here seems not good. I mean in 5 months it will become old. 
More then this it add maintenance on documentation.

What is needed here is a `changelog.md`  with versioning. 
Since we have already the rubygem Gemfile, we can add this, so an user can grep for the `new` changes.

https://github.com/openSUSE/gitarro/issues/156

Reply:
awesome, agree